README.md

```diff
@@ -24,7 +24,7 @@ A website with a calculator
 To get a local copy up and running follow this simple step:
 
 - Clone repository to your local machine: `www.github.com/gnwin/math-magicians.git`
-- Run `git checkout feature/events` to switch to develop branch
+- Run `git checkout feature/hooks` to switch to develop branch
```

Comment:
- [ ] You have some merging conflict, please resolve it. I think the conflict comes from readme file